platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerView.swift

```diff
@@ -105,20 +94,16 @@ public struct WysiwygComposerView: UIViewRepresentable {
                                       textView.logText,
                                       "Replacement: \"\(text)\""],
                                      functionName: #function)
-            return replaceText(textView.attributedText, range, text)
-        }
-
-        public func textViewDidChange(_ textView: UITextView) {
-            Logger.textView.logDebug([textView.logSelection,
-                                      textView.logText],
-                                     functionName: #function)
-            didUpdateText(textView)
+            return replaceText(textView, range, text)
         }
 
         public func textViewDidChangeSelection(_ textView: UITextView) {
             Logger.textView.logDebug([textView.logSelection],
                                      functionName: #function)
-            select(textView.attributedText, textView.selectedRange)
+            // Fixes long press delete issue
+            DispatchQueue.main.async {
+                self.select(textView.attributedText, textView.selectedRange)
+            }
```

Comment:
I'm a bit worried about this part, but I think it's pretty hard (impossible ?) to make UI tests that will check for cursor positions to make sure it is always correct

Reply:
sadly this is the only way I found to fix the long press delete skipping some text issue.
I tested the view myself and dispatching the selection on the next UI update seems to not affect the user experience in any way.
But yeah would be nice to have a test for this, but I have no idea how to write it.